### DISCUSSION TITLE
Minio Tenant - Web interface log out due to inactivity

### DISCUSSION THREAD

**Discussion by Shinlchi** [minio/minio#14896] [Q&A] [answered]
Hello, I met an issue with session expiry for Minio console Web UI.

I successfully bind Minio with my LDAP server thanks to environment variable and I want to set an session expiration to 15 minutes using the same mechanism.
Unfortunately I found out that `MINIO_IDENTITY_LDAP_STS_EXPIRY` was disabled since Jul,22 2021 according [to this modification](https://github.com/minio/minio/pull/12781) 

The previous post describe a redundancy of the parameter `MINIO_IDENTITY_LDAP_STS_EXPIRY` but I did not found any similar parameter to set an session timeout. 

Do you have any hint about a workaround on this topic ?

Best regards,

**Comment by harshavardhana** [comment, marked as answer]
you cannot increase this for the time being @Shinlchi

**Reply by huyz** [reply]
Will that capability ever be implemented?

**Reply by NicolasBitauldPRS** [reply]
Hello,
no news of this implementation ?
